src/shared/types.ts

```diff
@@ -34,27 +34,25 @@ interface Profile {
   timestamp: Date;
 }
 
-interface Reading {
-  id: numeric;
+export interface Reading {
+  id: string;
   meter: string;
-  temperature: numeric;
-  humidity: numeric;
-  pressure: numeric;
+  temperature: number;
+  humidity: number;
+  pressure: number;
   timestamp: Date;
 }
 
-interface Worker {
+export interface Worker {
   worker: string;
   nickname: string;
   timestamp: Date;
 }
 
-interface Zone {
-  id: numeric;
+export interface Zone {
+  id: string;
   nickname: string;
   parent: string;
   profile: string;
   timestamp: Date;
 }
-
-export = { Manufacturer, Device, Notification, Profile, Reading, Worker, Zone };
```

Comment:
Moved this file to `types.ts` so that other modules can import and use types where necessary